corehq/apps/performance_sms/forms.py

```diff
@@ -166,6 +166,9 @@ def __init__(self, domain, config, *args, **kwargs):
             *form_layout
         )
 
+    def clean_schedule(self):
+        return ScheduleConfiguration.wrap(self.cleaned_data['schedule'])
```

Comment:
nope, that's right and a good suggestion. https://github.com/dimagi/commcare-hq/pull/15049/commits/ac71e947ccd2a84a4614c620ef685eb09743c123